client/components/costume/EditCostume.js

```diff
@@ -17,13 +18,13 @@ class EditCostume extends Component {
   }
 
   async componentDidMount() {
-    await this.props.dispatchLoadSCostume(this.state.id);
-    await this.props.dispatchFetchCategories();
+    await this.props.dispatchLoadSCostume(this.state.costId);
```

Comment:
minor point but can this be called costumeId for consistency and to avoid confusion?